packages/boot/test/bootstrapTests/test-liquidation-concurrent-1.ts

```diff
@@ -0,0 +1,557 @@
+// @ts-check
+/** @file Bootstrap test of liquidation across multiple collaterals */
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { NonNullish } from '@agoric/assert';
+import { Offers } from '@agoric/inter-protocol/src/clientSupport.js';
+import process from 'process';
+import { TestFn } from 'ava';
+import {
+  LiquidationTestContext,
+  addSTARsCollateral,
+  likePayouts,
+  makeLiquidationTestContext,
+  scale6,
+} from './liquidation.ts';
+
+const test = anyTest as TestFn<LiquidationTestContext>;
+
+const atomSetup = {
```

Comment:
I'm not reviewing the arithmetic. I assume you'll sync with PM on the correctness per their spec